### ISSUE TITLE
anndata2ri import breaks hdf5r import

### ISSUE THREAD

**Issue by scottgigante-immunai** [theislab/anndata2ri#69]
If I import anndata2ri before I load hdf5r, then loading hdf5r throws an error.

MWE:

```python
import scipy.sparse
import rpy2.robjects as ro
ro.r("library('hdf5r')")
from anndata2ri import scipy2ri

scipy2ri.py2rpy(scipy.sparse.csr_matrix((100, 100)))
```

runs perfectly.

```python
import scipy.sparse
import rpy2.robjects as ro
from anndata2ri import scipy2ri
ro.r("library('hdf5r')")

scipy2ri.py2rpy(scipy.sparse.csr_matrix((100, 100)))
```

raises an error

```pytb
R[write to console]: Error: package or namespace load failed for ‘hdf5r’:
 .onLoad failed in loadNamespace() for 'hdf5r', details:
  call: fun(libname, pkgname)
  error: Error retrieving current error handler

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/conda/envs/analysis/lib/python3.8/site-packages/rpy2/robjects/__init__.py", line 438, in __call__
    res = self.eval(p)
  File "/opt/conda/envs/analysis/lib/python3.8/site-packages/rpy2/robjects/functions.py", line 198, in __call__
    return (super(SignatureTranslatedFunction, self)
  File "/opt/conda/envs/analysis/lib/python3.8/site-packages/rpy2/robjects/functions.py", line 125, in __call__
    res = super(Function, self).__call__(*new_args, **new_kwargs)
  File "/opt/conda/envs/analysis/lib/python3.8/site-packages/rpy2/rinterface_lib/conversion.py", line 45, in _
    cdata = function(*args, **kwargs)
  File "/opt/conda/envs/analysis/lib/python3.8/site-packages/rpy2/rinterface.py", line 677, in __call__
    raise embedded.RRuntimeError(_rinterface._geterrmessage())
rpy2.rinterface_lib.embedded.RRuntimeError: Error: package or namespace load failed for ‘hdf5r’:
 .onLoad failed in loadNamespace() for 'hdf5r', details:
  call: fun(libname, pkgname)
  error: Error retrieving current error handler
```

**Comment by scottgigante-immunai** [comment]
Package versions:
```
>>> anndata2ri.__version__
'0.0.0'
>>> rpy2.__version__
'3.4.2'
>>> ro.r('packageVersion("hdf5r")')[0]
<rpy2.robjects.vectors.IntVector object at 0x7fd2ddc665c0> [RTYPES.INTSXP]
R classes: ('integer',)
[1, 3, 3]
```

**Comment by scottgigante-immunai** [comment]
Solved by upgrading `h5py` from `2.10.0` to `3.6.0`.
